.changeset/mighty-things-brake.md

```diff
@@ -0,0 +1,6 @@
+---
+"@digdir/designsystemet-react": patch
+"@digdir/designsystemet-css": patch
+---
+
+Update `@u-elements/u-details` to `0.1.1`
```

Comment:
```suggestion
**Defailts**: Update `@u-elements/u-details` to `0.1.1`
```

Reply:
I suggest keeping the changelog for updating dependencies the same to be consistent (ie, #3426).